sotoki/sotoki.py

```diff
@@ -1256,7 +1260,12 @@ def create_zim(
 
     cmd = "zimwriterfs "
     if nopic:
```

Comment:
You should not use temporary FS here.
We just rename directory containing images temporary when making zim without image.